tools/myst/myst.edl

```diff
@@ -513,5 +513,13 @@ enclave
             [out] uint32_t* rbx,
             [out] uint32_t* rcx,
             [out] uint32_t* rdx);
+        
+        long myst_get_mempolicy_ocall(
+            [out] int* mode,
+            [out, size=maxnode] unsigned long* nodemask,
```

Comment:
`maxnode` indicates the num of bits within `nodemask` buffer. EDL syntax `size` is probably in bytes. Might need to convert `maxnode` in bits to a different parameter, for example, number of unsigned long elements in the `nodemask` buffer.

Reply:
I added a line maxnode/=8 to go around this in the caller function for this, but I changed that approach to add a different variable in my update, thanks